storage/helpers/customDatabaseRemover.go

```diff
@@ -0,0 +1,95 @@
+package helpers
+
+import (
+	"fmt"
+	"math/big"
+	"strings"
+
+	"github.com/ElrondNetwork/elrond-go/config"
+)
+
+const (
+	patternItemsSeparator = ","
+	divisibleByChar       = "%"
+)
+
+type customDatabaseRemover struct {
+	shouldRemoveHandler func(dbIdentifier string, epoch uint32) bool
```

Comment:
please refactor this. We do not need this handler, not even for testing purposes.

Reply:
okay, used a slice of divisible epochs